content/en/docs/concepts/services-networking/service.md

```diff
@@ -144,7 +144,7 @@ field.
 
 ### Port definitions {#field-spec-ports}
```

Comment:
```suggestion
### Port specifications {#field-spec-ports}
```
Hi @mboukhalfa, thanks for opening this PR. I agree that the sentence after the title is confusing. Even its title is too.  I assumed a resource called “Port” exists in Kubernetes due to the title “Port definitions”. 
How about you use “Port specifications”?
It indicates that a port is being specified. Since the documentation [associates](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/#configuration-files) a **definition** with a file having JSON or YAML format, it makes sense to say that its context is misused here. After all, a port is already a specification in a pod template.
I used a plural because multiple ports can be specified in a container.